### ISSUE TITLE
Displaying links to works within abstracts

### ISSUE THREAD

**Issue by nathangibson** [srophe/srophe-eXist-app#536]
Currently the XSLT does not render links to works that are inside abstracts. @davidamichelson how should we markup these references to works? For example, the abstract for http://wwwb.library.vanderbilt.edu/exist/apps/srophe/person/2784 is 

> Bishop of Iconium connected with the Cappadocian Fathers. Syriac translations of works attributed to him include an encomium on Basil, and a hagiographic work about Athanasius of Alexandria.

There are 3 options I know of:
- `<bibl>an encomium on Basil<ptr target="http://syriaca.org/work/382"></bibl>` Using this means writing some special XSLT that handles bibls in abstracts differently from bibls elsewhere.
- `<title ref="http://syriaca.org/work/382">an encomium on Basil</title>` But not all our references to works are titles, per se.
- `<ref target="http://syriaca.org/work/382">an encomium on Basil</ref>` Generic, but doesn't conflict with bibl.

**Comment by nathangibson** [comment]
FYI, option 1 is what I've been using so far.

**Comment by wsalesky** [comment]
I added a template for option 1. 

See:
http://wwwb.library.vanderbilt.edu/exist/apps/srophe/person/2784

**Comment by nathangibson** [comment]
Thanks! That looks great and does the trick for now. I guess we will still need to figure out whether we want to break TEI guidelines and put a ref on the bibl.

**Comment by davidamichelson** [comment]
Hi @wsalesky and pulling out LAWDI old man @paregorios and also @tacarlson into this discussion.

So following this discussion on the TEI-L it looks we might need to reconsider how we handle URIs in out various data templates:
https://listserv.brown.edu/archives/cgi-bin/wa?A2=ind1504&L=TEI-L&D=0&1=TEI-L&9=A&I=-3&J=on&d=No+Match%3BMatch%3BMatches&z=4&P=23902 

Thoughts:

There are two ways to mark a URI:
```<persName @ref> or <idno @type=”URI”>
<persName @ref=”http://syriaca.org/person/13”>Ephrem</persName>
<persName>Ephrem<idno type=“URI”>http://syriaca.org/person/13</idno></persName>```
Note: TEI:person is of limited use to us since it can only be used in a few places. Technically our assertions of the URI with persName are not semantically correct since we are indicating the person not the name, but this is not possible in TEI and our usage is a common work around.

As indicated in the discussion the second example actually fits LOD usage better since it points to a URI for the person and not a document defining the person. Switching to this method would also help with uniformity on this particular issue (see above) because we could mark up TEI:bibl (which does not allow att.canonical: ```@ref```) in this way:

```<bibl>an encomium on Basil<idno @type="URI">http://syriaca.org/work/382</idno></bibl>

Using TEI:idno instead of @ref  would be a major change across our templates but could be done in a script.

Thoughts?

**Comment by nathangibson** [comment]
@wsalesky @dlschwartz Do you remember how we resolved this? I think we decided to continue using `@ref` on the persNames and placeNames, but did we decide for sure to use `<bibl>...<ptr>` for references to works?

**Comment by wsalesky** [comment]
@nathangibson I do not think we did resolve it. I think we are leaning toward tei:idno, but I do not know that we have made this a policy, and we have not made data changes.

**Comment by nathangibson** [comment]
Note that we may currently be using two different formats (title with ref and bibl with ptr) in our data. Once we decide on a standard we should standardize the data.
